README.md

```diff
@@ -3,406 +3,224 @@
-* [Author](#author)
+* [Creators](#creators)
 
 ## Features
 
-### Easy Install
-
-Everything is included in a single module installed with one command.
-
-```bash
-npm install -g testcafe
-```
-
-No native parts to compile, no browsers plugins to install.
-
-### Complete Test Harness
-
-TestCafe automatically starts browsers, runs tests and gathers results. You only type a single command to begin testing.
-
-```bash
-testcafe chrome tests/
-```
-
-When testing is finished, TestCafe aggregates test results from different browsers and outputs them into one comprehensive report.
-
-### Write Test Code in Modern JavaScript and TypeScript
-
-TestCafe allows you to write tests using [TypeScript](https://devexpress.github.io/testcafe/documentation/test-api/typescript-support.html) or JavaScript (with its modern features like `async/await`).
-
-[Test API](https://devexpress.github.io/testcafe/documentation/test-api/index.html) consists of over two dozen methods that can emulate all actions one could possibly do with a webpage.
-Chained syntax allows for code that is easy to write and read.
-
-```js
-fixture `Example page`
-    .page `https://devexpress.github.io/testcafe/example`;
-
-test('Emulate user actions and perform a verification', async t => {
-    await t
-        .setNativeDialogHandler(() => true)
-        .click('#populate')
-        .click('#submit-button');
-
-    const location = await t.eval(() => window.location);
-
-    await t.expect(location.pathname).eql('/testcafe/example/thank-you.html');
-});
-```
-
-By using TypeScript to write your TestCafe tests, you get the advantages of strongly-typed languages such as: rich coding assistance, painless scalability, check-as-you-type code verification, and much more.
-
-To make debugging easier, TestCafe automatically generates source maps.
-To debug your test code, start a debugging session in an IDE that supports source maps.
+**Stable tests and no manual timeouts**<br/>
+TestCafe provides built-in assertions which wait until page elements load.
+You can change the maximum wait time.
+If elements load faster, tests skip the timeout and continue.
 
-### Flexible Selector System
+**Latest JS and TypeScript support**<br/>
+TestCafe supports the latest JavaScript features, including ES6, ES7 (for example, async/await).
+You can also [use TypeScript](https://devexpress.github.io/testcafe/documentation/test-api/typescript-support.html)
+if you prefer a strongly typed language.
 
-TestCafe supports a flexible selector system that provides API with rich capabilities for writing test scripts.
-You can access webpage elements in different ways using [selectors](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). For example, you can use one of the following selector types:
+**Detects JS errors in your code**<br/>
+TestCafe reports JS errors that it finds on the webpage.
+Tests automatically fail because of that.
+However, you can disable this.
```

Comment:
A link to the related option?

However, you can [disable](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-e---skip-js-errors) this

Reply:
If we have a good search in our docs (I hope we do), then it should be easy to find.

Each link we include should have a purpose. If it's an article, then the more links to our site - the better. But on a landing page I don't see such need. It's not a common question like "how come you don't use selenium".

Or is there something special about disabling this feature?

Reply:
I agree. Nothing special there